adr/0012-use-networkManager-for-network-requests.md

```diff
@@ -0,0 +1,18 @@
+# Using NetworkManager instead of Fetchable protocol
+
+* Status: accepted
+* Deciders: iOS Team
+* Date: 2020-12-30
+
+## Context
+
+Our team created NetworkManager for network requests. For consistency we should replace and use NetworkManager instead of Fetchable protocol
+
+## Decision
+
+Every new interactors must use NetworkManager.
```

Comment:
i think you should add info which is every new endpoint should use networker system for old interactors 
```suggestion
Every new interactors must use NetworkManager. Also every new endpoints should use new networker approach
```